src/main/java/reposense/system/LogsManager.java

```diff
@@ -88,30 +91,75 @@ private static void removeHandlers(Logger logger) {
      * Creates {@link FileHandler} if it is null.
      */
     private static void addFileHandler(Logger logger) {
-        Path path = logFolderLocation.resolve(LOG_FOLDER_NAME);
+        Path path = TEMP_LOG_DIR.resolve(LOG_FOLDER_NAME);
 
         try {
-            if (!Files.exists(path)) {
+            boolean isFileExist = Files.exists(path);
+            if (!isFileExist) {
```

Comment:
should this and line 107 be combined? since this actually creates the log temp folder